rules_gapic/bazel/src/main/java/com/google/api/codegen/bazel/BazelBuildFileView.java

```diff
@@ -96,7 +96,7 @@ private String assembleGoImportPath(boolean isCloud, String protoPkg, String goP
     String goImport = "";
     if (isCloud) {
       goImport = "cloud.google.com/go/";
-      goPkg = goPkg.replaceFirst("v(.+);", "apiv$1;");
+      goPkg = goPkg.replaceFirst("\\/v([a-z1-9]+);", "\\/apiv$1;");
```

Comment:
what about 0? Version probably can' start with 0, but v10 is theoretically possible.

Reply:
Good idea. Done!